zoloto/cameras/base.py

```diff
@@ -103,18 +123,18 @@ def get_marker_size(id_and_corners):
                     int(marker_id), corners, size, calibration_params, tvec[0], rvec[0]
                 )
 
-    def get_visible_markers(self, *, frame=None):
+    def get_visible_markers(self, *, frame: ndarray = None) -> List[int]:
         ids, _ = self._get_ids_and_corners(frame)
         return ids
 
-    def close(self):
+    def close(self) -> None:
         pass
 
-    def __enter__(self):
+    def __enter__(self) -> "BaseCamera":
```

Comment:
Does this deal with making the type more specific on subclasses?

Reply:
a85c877